Libplanet/PoS/Util/AddressComparer.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Libplanet.PoS
+{
+    internal class AddressComparer : IComparer<Address>
+    {
+        public int Compare(Address x, Address y)
+        {
+            int result
+                = (new BigInteger(x.ToByteArray())
+                - new BigInteger(y.ToByteArray())).Sign;
+
+            return -result;
+        }
+    }
```

Comment:
How about using [partial class](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods)?

Reply:
Not clearly understood. You mean make class nested?

Reply:
NVM, I'm confused 😵‍💫